java/vector/src/test/java/org/apache/arrow/vector/types/pojo/TestExtensionType.java

```diff
@@ -278,7 +278,7 @@ public void testVectorCompare() {
     }
   }
 
-  static class UuidType extends ExtensionType {
+  public static class UuidType extends ExtensionType {
```

Comment:
nit: should we add docs, since it's going to be public?

Reply:
This is still under the test directory.